processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByMergingQueryRunnerV2.java

```diff
@@ -148,6 +134,11 @@ public GroupByMergingQueryRunnerV2(
         String.format("druid-groupBy-%s_%s", UUID.randomUUID(), query.getId())
     );
 
+    final LimitedTemporaryStorage temporaryStorage = new LimitedTemporaryStorage(
```

Comment:
Any reason this is moved from where it was originally? It would be better to defer creation until the Sequence has started accumulating (see #3296)